A/AmazonCACerts/bundled/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2020 Invenia Technical Computing Corporation
```

Comment:
I'm a bit confused by this line: is this certificate produced by Invenia?

Reply:
Ah, I think I got confused here reading https://juliapackaging.github.io/BinaryBuilder.jl/dev/jll/ :
> LICENSE, a file stating the license of the JLL package. Note that this may differ from the license of the library it wraps, which is instead shipped inside the tarballs;

But now I realise this is for the `NAME_jll.jl` repo, which I understand will be generated after my PR gets merged.

I'll try to find the license for that AWS certificate and add it.